docs/learn/learn-staking-advanced.md

```diff
@@ -111,62 +111,75 @@ On Polkadot and Kusama, the instance of the pallet
+automatically in the 1st bag because the total number of bonded tokens will be outside the range of
+the 2nd bag (see figure below). That node with now 21 DOT will be put at the tail of the 1st bag
+with the possibility to manually put itself in front of "older" nodes with less than 21 DOT (if
+there are any).
+
+![bags list example 1](../assets/bags-list-example-1.png)
+
+If one decides to automatically send staking rewards to the stash account and bond them (i.e.
+compound staking), the position within a bag changes over time and this may result in a change of
+bag. Any nominator within a specific bag can submit the permissionless extrinsic `rebag` within the
+`voterList` pallet instance to update the positions of those accounts that do not belong to their
+bag, and place them in the correct one. Actions like bonding/unbonding tokens automatically rebags
+the nominator node, but events like staking rewards/slashing do not. See the
+[bags-list](learn-nominator.md#bags-list) section for more information.
+
+The bags-list is capable of including an unlimited number of nodes, subject to the chain's runtime
+storage. In the current staking system configuration, the bags list keeps
 {{ polkadot: <RPC network="polkadot" path="query.staking.maxNominatorsCount" defaultValue={50000}/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="query.staking.maxNominatorsCount" defaultValue={20000}/> :kusama }}
 nomination intents, of which, at most
 {{ polkadot: <RPC network="polkadot" path="query.electionProviderMultiPhase.maxElectingVoters" defaultValue={22500}/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.electionProviderMultiPhase.maxElectingVoters" defaultValue={20000}/> :kusama }}
 come out as the electing nominators. See
 [Staking Election Stages](learn-nominator.md#staking-election-stages) section for more info.
 
+This means that only a portion of the nomination intents is kept. Once the nomination period ends,
+the NPoS election mechanism takes all nomination intents and their associated votes as input, and it
+outputs a set of validators. The bags are iterated from the most staked to the least staked. If the
+accounts are not appropriately sorted, this could leave the last touched bag to only be partially
+iterated. Thus, in some edge cases, the order of the members within a bag is important. Continuing
+with the example used in the previous figures, there are 8 nomination intents of which only 7 will
+be kept. If the bags list stays semi-sorted (i.e. no accounts call the `putInFrontOf` and `rebag`
+extrinsics), the nomination of the node with 8 DOT in the 3rd bag will not be considered while that
+of the preceding node with 5 DOT will. Nomination of the node with 8 DOT will be kept only if it
+will put itself in front of the one with 5 DOT. Note how the nomination of the node with 19 DOT in
+the 3rd bag will be considered regardless of changing its position inside the bag.
```

Comment:
Isn't the 19 DOT in the second bag? It might be worth adding the numbers to the bags, as we are always referring to them that way

Reply:
right, thanks!